src/time-component.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from "react";
+
+export interface InputTimeProp {
+  value: string;
+}
+
+export class InputTime extends React.Component<InputTimeProp> {
+  constructor() {
+    super();
+
+    this.state = {
+      time: "12:34"
+    };
+
+    this.onUpdate = this.onUpdate.bind(this);
+  }
+
+  onUpdate() {
+    this.setState({ time: event.target.value });
+  }
+
+  render() {
+    return (
+      <form>
+        <label>
+          Time:
+          <input
+            type="time"
+            value={this.state.value}
+            onChange={this.onUpdate}
```

Comment:
Following the specs, the component must send the update to the parent component. You will have to pass a function in the props that will be used as listener. You can find more details in the [ReactJS documentation](https://reactjs.org/docs/faq-functions.html)